gddebug.cc

```diff
@@ -48,7 +48,7 @@ QTextCodec *localeCodec = 0;
     QTextCodec::setCodecForLocale( utf8Codec );
   }
 
-  qDebug( "%s", QString().vsprintf( msg, ap ).toUtf8().data() );
+  qDebug( "%s", QString().vasprintf( msg, ap ).toUtf8().data() );
```

Comment:
`vasprintf` is a [GNU extension](https://stackoverflow.com/questions/12746885/why-use-asprintf-instead-of-sprintf). Not portable.